pkg/common/volume_operations.go

```diff
@@ -172,10 +193,24 @@ func (vo *VolumeOperationsImpl) CreateVolume(ctx context.Context, v api.Volume)
 		if err = vo.k8sClient.UpdateCRWithAttempts(ctxWithID, ac, 5); err != nil {
 			ll.Errorf("Unable to set size for AC %s to %d, error: %v", ac.Name, ac.Spec.Size, err)
 		}
+		if vo.acReservationEnabled {
+			resHelper := capacityplanner.NewReservationHelper(vo.log, vo.k8sClient, capReader, resReader)
+			if err = resHelper.ReleaseReservation(ctxWithID, origAC, ac); err != nil {
+				ll.Errorf("Unable to remove ACR reservation for AC %s, error: %v", ac.Name, err)
+			}
+		}
 	}
 	return &volumeCR.Spec, nil
 }
 
+func (vo *VolumeOperationsImpl) createCapacityManager(capReader capacityplanner.CapacityReader,
```

Comment:
Please check for volume binding mode here as well or create issue to address problem with _Immediate_ later.

Reply:
Issue created https://github.com/dell/csi-baremetal/issues/127